core/src/main/resources/error/error-classes.json

```diff
@@ -757,6 +757,16 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_LOG_K" : {
```

Comment:
I propose to rename this to `LG` as opposed to `LOG`. "Log" is typically base 10, and "lg" is typically base 2. In this case we have a base-2 logarithm.

Reply:
Sounds good, done.